db/schema.rb

```diff
@@ -443,8 +444,8 @@
   end
 
   create_table "tasks", id: :serial, force: :cascade do |t|
-    t.integer "owner_id"
     t.string "owner_type"
+    t.integer "owner_id"
```

Comment:
Hrm, couple of unrelated changes in here.  Are you working off a prod backup?

Reply:
it just changed the order of the columns, not the type or anything - is that a problem?

Reply:
(I was not working off a copy of prod when I ran these migrations.)

Reply:
Not a problem - it'll just flicker back the next time someone runs a migration locally.